posthog/hogql/ast.py

```diff
@@ -631,6 +631,18 @@ class CompareOperationOp(StrEnum):
     NotIRegex = "!~*"
 
 
+NEGATED_COMPARE_OPS: list[CompareOperationOp] = [
```

Comment:
Bit of a nitpick, but wouldn't this be marginally faster as a `set`/`frozenset` for it's use in `where_clause_extractor`? (I didn't actually benchmark this myself.)